### ISSUE TITLE
Selection area does not move with the image

### ISSUE THREAD

**Issue by ratnazone** [odyniec/imgareaselect#30]
Test on - IE 9, chrome (v 23.0)
1. Select an area in the image
2. Move the image up/down (say by adding a div element before the image)
3. image will move down but the selection area will not

**Comment by tedtkang** [comment]
Has anyone made any headway into this issue?  I'm trying to resolve or figure out a fix.  I tried the parent option and gave the img I needed an outer div, but that didn't seem to work.

**Comment by tedtkang** [comment]
nevermind.  i got this working.  the parent option works out fine as long as you make your outer div position: relative.

**Comment by odyniec** [comment]
Yeah, the `parent` option, if used properly, should resolve any positioning problem.

@ratnazone, if you're still experiencing your issue, please show me the actual HTML/JS code that demonstrates it, or better yet, a live demo.

(And sorry for the awfully late response, unfortunately I always have too little time for my personal projects _sigh_)
